resources/assets/lib/beatmap-discussions/editor-discussion-component.tsx

```diff
@@ -26,7 +26,9 @@ export default class EditorDiscussionComponent extends React.Component<Props> {
 
   componentDidMount = () => {
     // reset timestamp to null on clone
-    Transforms.setNodes(this.context, {timestamp: null}, {at: this.path()});
+    if (this.editable()) {
```

Comment:
looks like it is reset again when navigated using keyboard (by pressing left arrow key several times)

![](https://s.myconan.net/2020-07/2020-07-22_15-21-16.gif)